<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CampaignProperties</class>
 <widget class="QDialog" name="CampaignProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="m_tabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="dicealias">
      <attribute name="title">
       <string>Dice Alias</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,0,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Dice Command</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_cmdDiceEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="m_testPushButton">
             <property name="text">
              <string>Test</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Converted</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_convertedCmdEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QTableView" name="m_tableViewAlias">
           <property name="editTriggers">
            <set>QAbstractItemView::AllEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QToolButton" name="m_addDiceAliasAct">
             <property name="toolTip">
              <string>Add</string>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset theme="add">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_delDiceAliasAct">
             <property name="toolTip">
              <string>Delete</string>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
             <property name="icon">
              <iconset theme="remove">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_topDiceAliasAct">
             <property name="toolTip">
              <string>Top</string>
             </property>
             <property name="text">
              <string>Top</string>
             </property>
             <property name="icon">
              <iconset theme="go-top">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_upDiceAliasAct">
             <property name="toolTip">
              <string>Up</string>
             </property>
             <property name="text">
              <string>Up</string>
             </property>
             <property name="icon">
              <iconset theme="go-up">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_downDiceAliasAct">
             <property name="toolTip">
              <string>Down</string>
             </property>
             <property name="text">
              <string>Down</string>
             </property>
             <property name="icon">
              <iconset theme="go-down">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_bottomDiceAliasAct">
             <property name="toolTip">
              <string>Bottom</string>
             </property>
             <property name="text">
              <string>Bottom</string>
             </property>
             <property name="icon">
              <iconset theme="go-bottom">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Character States</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTableView" name="m_stateView">
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolButton" name="m_addCharacterStateAct">
             <property name="toolTip">
              <string>Add</string>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset theme="add">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_delCharceterStateAct">
             <property name="toolTip">
              <string>Delete</string>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
             <property name="icon">
              <iconset theme="remove">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_topCharacterStateAct">
             <property name="toolTip">
              <string>Top</string>
             </property>
             <property name="text">
              <string>Top</string>
             </property>
             <property name="icon">
              <iconset theme="go-top">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_upCharacterStateAct">
             <property name="toolTip">
              <string>Up</string>
             </property>
             <property name="text">
              <string>Up</string>
             </property>
             <property name="icon">
              <iconset theme="go-up">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_downCharacterStateAct">
             <property name="toolTip">
              <string>Down</string>
             </property>
             <property name="text">
              <string>Down</string>
             </property>
             <property name="icon">
              <iconset theme="go-down">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_bottomCharacterStateAct">
             <property name="toolTip">
              <string>Bottom</string>
             </property>
             <property name="text">
              <string>Bottom</string>
             </property>
             <property name="icon">
              <iconset theme="go-bottom">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="m_nameEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Root Directory:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="FileDirChooser" name="m_rootDir" native="true"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>File count:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="m_fileCountLbl">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Disk usage:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="m_diskUsageLbl">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Theme*:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="m_currentTheme"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>497</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileDirChooser</class>
   <extends>QWidget</extends>
   <header location="global">customs/filedirchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CampaignProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CampaignProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
